doc/internal/man3/OPTIONS.pod

```diff
@@ -214,34 +227,40 @@ end of arguments has been reached. This is typically done like this:
         }
     }
 
-The opt_help() function takes a list of option definitions and prints a
-nicely-formatted output.
+Within the option parsing loop, the following functions may be called.
+
+The opt_flag() fuction returns the most recent option name
+including the preceding C<->.
 
-Within the option parsing loop, opt_flag() returns the option,
-without any leading hyphens. The opt_arg() function returns
-the option's value, if there is one.
+The opt_arg() function returns the option's argument value, if there is one.
 
+The opt_unknown() function returns the unknown option.
 In an option list, there can be at most one option with the empty string.
 This is a "wildcard" or "unknown" option. For example, it allows an
-option to be be taken as digest algorithm, like C<-sha1>. The
+option to be be taken as digest algorithm, like B<-sha1>. The
```

Comment:
Unsure about this change. As it includes the `-` I'd prefer using `C<>`

Reply:
Reverted.